pages/common/git-bundle.md

```diff
@@ -0,0 +1,32 @@
+# git bundle
+
+> Package objects and references in an archive.
+> More information: <https://git-scm.com/docs/git-bundle>.
+
+- Create a bundle file that contains all objects and references of a specific branch:
+
+`git bundle create {{path/to/file.bundle}} {{branch_name}}`
+
+- Create a bundle file of all branches:
+
+`git bundle create {{path/to/file.bundle}} --all`
+
+- Create a bundle file of the last 5 commits of the current branch:
+
+`git bundle create {{path/to/file.bundle}} -{{5}} {{HEAD}}`
+
+- Create a bundle file of the latest 7 days:
+
+`git bundle create {{path/to/file.bundle}} --since={{7.days}} {{HEAD}}`
+
+- Verify that a bundle file is valid and can be applied to the current repository:
+
+`git bundle verify {{path/to/file.bundle}}`
+
+- Print to standard output the list of references contained in a bundle:
```

Comment:
Another really minor point, but perhaps this will read better?
```suggestion
- Print to the standard output a list of references contained in a bundle:
```

Reply:
Another pages use `stdout` in the example description. We should take 1 form.

Reply:
I agree with "`the` standard output", but I'm not sure about using "`a` list of references". Is not any list of references in the bundle, but the sole and only :)

Reply:
> Another pages use `stdout` in the example description. We should take 1 form.

We might consider tackling this on a broader scope, checking all pages